HW1/P5/P5.txt

```diff
@@ -1 +1,20 @@
 # Your discussion here
+
+Shortest paths from Kevin Bacon -> Harvard University 
+
+[2729536, 5114592, 2729536, 3229511, 2729536, 4625677, 2729536, 2578703, 2729536, 1124925]
+
+[(1124925, u'College_Bowl'), (2578703, u'John_Lithgow'), (3229511, u'Marisa_Silver'), (4625677, u'Six_degrees_of_separation'), (5114592, u'Time_(magazine)')]
+
+
+Shortest paths from Harvard University -> Kevin Bacon 
+120 such paths, which are listed below in numerical identifiers and then by name. 
+
```

Comment:
Hey Patrick, sorry the meaning of the things below is that the first identifier is Kevin Bacon, the second and the third are the nodes BEFORE Harvard. So each has 2 intermediary steps.